src/CodeGen_PTX_Dev.cpp

```diff
@@ -637,6 +637,16 @@ vector<char> CodeGen_PTX_Dev::compile_to_src() {
     raw_svector_ostream ostream(outstr);
     ostream.SetUnbuffered();
 
+    // NOTE: use of the "legacy" PassManager here is still required; it is deprecated
+    // for optimization, but is still the only complete API for codegen as of work-in-progress
+    // LLVM14. At the time of this comment, there is no firm plan as to when codegen will
```

Comment:
> At the time of this comment

I know I can check the `git blame`, but it might be worth jotting down (12/10/21) anyway.

Reply:
Done